generators/server/templates/_build.gradle

```diff
@@ -102,6 +102,11 @@ description = ''
 
 configurations {
     providedRuntime
+    <%_ if (applicationType == 'microservice' || applicationType == 'gateway' || applicationType === 'uaa') { _%>
+    // netty's native is pulled by spring-cloud-starter-ribbon, but is useless unless you explicitly add the native binary dependency.
+    // Having it in the classpath without the binary can cause warnings
+    all*.exclude group: 'io.netty', module: 'netty-transport-native-epoll'
```

Comment:
The warning only shows up with the Cassandra java driver, but this library is not the implementation used by Netty unlike the user explicitly add another lib with the binary.

Like this for linux:
```
    <dependency>
      <groupId>io.netty</groupId>
      <artifactId>netty-transport-native-epoll</artifactId>
      <version>${project.version}</version>
      <classifier>linux-x86_64</classifier>
    </dependency>
```

So I think it's better to remove it from the classpath, even if Cassandra is not present, and let the user decide if he really wants to activate the native support.
In that case he will add the 2 dependencies.